.travis.yml

```diff
@@ -56,6 +52,7 @@ jobs:
 branches:
   only:
     - master
+    - bias_correction
```

Comment:
Why do we want to limit Travis-CI to only `master` branch, then we have to add each dev branch manually, like this `bias_correction` branch? @Zeitsperre (b5a356609228a44a04928f91af5e5a76773546ef)

Reply:
This can be removed. It was placed in order to trigger the bias_correction branch that PRs were merging to in a few instances. The reason why I set only to master is that otherwise we tend to get two duplicate sets of builds; one that is the pushes set against master and another that is the pushes set against the last version of master it stemmed from.

Since we only care about the current state of master, as all pushes must integrate these changes, there's nothing to gain except more redundant build checks.

Reply:
I think the redundant build checks on `master` branch will happen only once, when the PR is merged to `master`.

Whereas instructions to build only the `master` branch will not test any commits on any devel branches and force all devel branches to hardcode the devel branch name like we just saw.

I think having the redundant build on `master` is a less bad of the 2 problems.  Wonder what's your opinion about removing this "build only `master` branch" instruction for all birds?